gubernator/view_build.py

```diff
@@ -143,23 +141,40 @@ def build_details(build_dir):
     return started, finished, parser.get_results()
 
 
-def parse_pr_path(prefix):
-    for org, path in view_base.PR_PREFIX.items():
-        if not prefix.startswith(path):
-            continue
-        pr = os.path.basename(prefix)
-        repo = os.path.basename(os.path.dirname(prefix))
-        if '_' in repo and not repo.startswith(org):
-            continue
-        if org == 'kubernetes' and repo == 'pull':
-            return pr, '', 'kubernetes/kubernetes'
-        pr_path = repo.replace('_', '/')
-        if '_' not in repo:
-            repo = '%s/%s' % (org, repo)
-        else:
-            repo = pr_path
-        return pr, pr_path + '/', repo
-    return None, None, None
+def parse_pr_path(gcs_path, pull_prefix, default_org, default_repo):
```

Comment:
I'd prefer leaving default_org/default_repo as kubernetes/kubernetes here, and move toward deprecating it.

The complicated path nonsense that we implemented for backwards compatibility as we added more repos turned out to be a mistake, as can be seen by the complexity that it causes throughout.

Reply:
I see the argument for that approach, one issue though is that we've used a short-form format for GCS artifacts in Origin. I'd rather we did it this way and slowly pushed everyone forward together. WDYT?

Reply:
Oh, if you're already using short-form then this makes sense.